src/rgw/rgw_cr_rados.cc

```diff
@@ -222,9 +222,6 @@ RGWRadosSetOmapKeysCR::RGWRadosSetOmapKeysCR(RGWRados *_store,
 
 RGWRadosSetOmapKeysCR::~RGWRadosSetOmapKeysCR()
 {
-  if (cn) {
-    cn->put();
-  }
```

Comment:
thanks. can you remove these destructors now that they're empty? (`~RGWRadosTimelogAddCR()` and `~RGWRadosTimelogTrimCR()` also)